trajpy/trajpy.py

```diff
@@ -63,30 +65,34 @@ def compute_features(self):
 
         :return features: return the values of the features as a string.
         """
-        self.msd_ta = self.msd_time_averaged(self._r,
-                                             np.arange(len(self._r)))
+        self.msd_ta = self.msd_time_averaged(self._r,np.arange(len(self._r)))
         self.msd_ea = self.msd_ensemble_averaged(self._r)
         self.msd_ratio = self.msd_ratio_(self.msd_ta, n1=2, n2=10)
         self.anomalous_exponent = self.anomalous_exponent_(self.msd_ea, self._t)
         self.fractal_dimension, self._r0 = self.fractal_dimension_(self._r)
 
-        self.gyration_radius = self.gyration_radius_(self._r)
-        self.eigenvalues, self.eigenvectors = np.linalg.eig(self.gyration_radius)
-        self.eigenvalues[::-1].sort()  # the eigenvalues must be in the descending order
-        self._idx = self.eigenvalues.argsort()[::-1]  # getting the position of the principal eigenvector
-        self.kurtosis = self.kurtosis_(self._r, self.eigenvectors[self._idx[0]])
-        self.anisotropy = self.anisotropy_(self.eigenvalues)
+        self.gyration_radius = list(self.gyration_radius_(self._r).values())[0]
```

Comment:
Why did you use the list() function here?

Reply:
The gyration_radius function now returns the gyration tensor itself, the eigenvalues and eigenvectors in a dictionary. So to get the values of the dictionary, I gotta use the values() function and put it on a list so that I can get the values in order... I tried to set it as an numpy array but the values() function does not work that way.

Reply:
The gyration_radius function now returns the gyration tensor itself, the eigenvalues and eigenvectors in a dictionary. So to get the values of the dictionary, I gotta use the values() function and put it on a list so that I can get the values in order... I tried to set it as an numpy array but the values() function does not work that way.